doc/release-notes/release-8.sgml.in

```diff
@@ -46,6 +46,10 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 	<p>Removed <em>ntlm_smb_lm_auth</em> NTLM authentication helper.
 	   Use the <em>ntlm_auth</em> helper from the Samba project instead.
 
+	<p>Fixed <em>digest_edirectory_auth</em> authentication helper. LDAPS
+		activation and referrals were broken, and memory could leak.
+		LDAPS now negotiates correctly, referrals honor configuration
```

Comment:
If nobody has actually tested that "LDAPS now negotiates correctly", then I would downplay that claim. For example:

```suggestion
		LDAPS may now negotiate correctly, referrals honor configuration
```

or

```suggestion
		LDAPS activation bug is now fixed, referrals honor configuration
```

I do not insist on addressing this (conditional) change request because I do not know whether its precondition is true.

@MegaManSec, did you see this helper successfully performing LDAPS negotiations in a Squid-driven test?